apps/prairielearn/src/ee/auth/saml/index.ts

```diff
@@ -12,7 +15,9 @@ export async function getSamlOptions({
   strictMode: boolean;
 }): Promise<SamlConfig> {
   const samlProvider = await getInstitutionSamlProvider(institution_id);
-  if (!samlProvider) throw new Error('No SAML provider found for given institution');
+  const InstitutionProvider = await getInstitutionAuthenticationProviders(institution_id);
+  if (!samlProvider || !InstitutionProvider.some((p) => p.name === 'SAML'))
+    throw new Error('No SAML provider found for given institution');
```

Comment:
Since this has wrapped to a new line, let's place the body of the if statement in a block with curly braces.